### ISSUE TITLE
Checkboxes are misaligned when used in a list

### ISSUE THREAD

**Issue by k-paxian** [Vonage/vivid-react#45]
```jsx
<VwcList>
      <VwcCheckListItem>foo</VwcCheckListItem>
      <VwcCheckListItem>bar</VwcCheckListItem>
</VwcList>
```

![image](https://user-images.githubusercontent.com/18071010/175791402-94fade72-ab95-4a90-8746-a71d773014f0.png)

Originated from:
https://vonage.slack.com/archives/C013F0YKH99/p1655985327387479

**Comment by k-paxian** [comment]
Post mortem

`@vonage/vwc-list/vwc-check-list-item` was not tested in isolation(mainly because of storybook environment, which is importing almost all vivid elements in a bulk it works properly there), it uses material checkbox under the hood, but means to use vwc-checkbox but **doesn't import the element explicitly**.

Since those extra styles are a part of that import we have broken expectations when this checklist is used in isolation

**Comment by k-paxian** [comment]
fixed in aca4618368043b902b5c66da473ab71b814f6ac1
